corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-{% if user.is_authenticated %}
+{% if request.user.is_authenticated %}
```

Comment:
@biyeun 
Some of the views override `user` with `couch_user`, so `user.is_authenticated` evaluates to null and this modal never gets filled. I would also prefer removing the check altogether from this modal-template and put that condition in `base.html` instead. But I will look for your inputs.